srcpkgs/PrusaSlicer/patches/cross.patch

```diff
@@ -1,11 +0,0 @@
---- a/src/avrdude/CMakeLists.txt	2020-06-04 09:19:26.017251215 +0200
-+++ b/src/avrdude/CMakeLists.txt	2020-06-04 09:19:36.468250771 +0200
-@@ -83,7 +83,7 @@ add_executable(avrdude-conf-gen conf-gen
- add_custom_command(
-     DEPENDS avrdude-conf-gen ${CMAKE_CURRENT_SOURCE_DIR}/avrdude-slic3r.conf
-     OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/avrdude-slic3r.conf.h
--    COMMAND $<TARGET_FILE:avrdude-conf-gen> avrdude-slic3r.conf avrdude_slic3r_conf ${CMAKE_CURRENT_BINARY_DIR}/avrdude-slic3r.conf.h
-+    COMMAND avrdude-conf-gen avrdude-slic3r.conf avrdude_slic3r_conf ${CMAKE_CURRENT_BINARY_DIR}/avrdude-slic3r.conf.h
-     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
- )
- 
```

Comment:
The code this file is patching no longer seems to exist.

I assume removing this is OK since `nocross` is set anyways.